src/theory/quantifiers/sygus/term_database_sygus.h

```diff
@@ -69,10 +69,8 @@ class TermDbSygus {
    * (see d_enum_to_active_guard),
    * useSymbolicCons : whether we want model values for e to include symbolic
    * constructors like the "any constant" variable.
-   * isVarAgnostic : if this flag is true, the enumerator will only generate
-   * values whose variables are in canonical order (for example, only x1-x2
-   * and not x2-x1 will be generated, assuming x1 and x2 are in the same
-   * "subclass", see getSubclassForVar).
+   * isActiveGen : if this flag is true, the enumerator will be
+   * actively-generated based on the mode specified by --sygus-active-gen.
```

Comment:
maybe keep the example? It's good to have some examples of what active gen enums do.